src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureArea {
+    double getArea();
+
+    void messageInfo();
```

Comment:
let's create a separate interface Drawable with method draw()
and make Figure class implement 2 interfaces, because the purposes of the interfaces are different: 1st calculates the area, second is printing out whole information